src/rpc/evo.cpp

```diff
@@ -87,14 +87,25 @@ static RPCArg GetRpcArg(const std::string& strParamName)
         {"operatorPubKey_register",
             {"operatorPubKey_register", RPCArg::Type::STR, RPCArg::Optional::NO,
                 "The operator BLS public key. The BLS private key does not have to be known.\n"
-                "It has to match the BLS private key which is later used when operating the masternode."}
+                "It has to match the BLS private key which is later used when operating the masternode.\n"}
```

Comment:
Looks like this adds an extra blank line.
```suggestion
                "It has to match the BLS private key which is later used when operating the masternode."}
```